test/e2e/common/node/pods.go

```diff
@@ -866,6 +866,8 @@ var _ = SIGDescribe("Pods", func() {
 				}}, metav1.CreateOptions{})
 			framework.ExpectNoError(err, "failed to create pod")
 			framework.Logf("created %v", podTestName)
+			framework.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(f.ClientSet, podTestName, f.Namespace.Name))
```

Comment:
Better to use `e2epod.WaitForsRunningReady` (https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/wait.go#L107) to wait for all 3 pods in parallel.

Reply:
Done